Bottom: 2b6b85a7dd773fd1d79baffa49cfeb9677be1915
Top:    a5357b671a680a91831aa153a0ddd5db2e156cd8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:01:08 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
